text/0000-compiler-team-trusted-contrib-maintainer.md

```diff
@@ -0,0 +1,389 @@
+*crater* permissions; *organisation membership*; and *dev desktop* access.
+Trusted contributors are considered members of the Rust project as a whole, and
+are automatically eligible for any benefits that incurs (e.g. invitations to
+meetups of project members). As representatives of the Rust project, trusted
+contributors are expected to obey not just the letter of the
+[Code of Conduct][coc] but its spirit.
+
+Trusted contributors can choose to take on additional responsibilities, such as
+those listed in the [responsibilities][responsibilities] section. Participating
+in the team's review rotation is encouraged.
+
+If a trusted contributor becomes inactive (the contributor's prior contributions
+and other interactions with the project cease) for longer than a year, the
+trusted contributor will be moved into alumni status. At any point in the
+future, they can ask to be re-instated at the trusted contributor level if they
+desire.
+
+[Compiler Development Guide]: https://rustc-dev-guide.rust-lang.org/
+
+Team Members
+------------
+[team-members]: #team-members
+
+Trusted contributors are eligible to become team members after they have
+continued to contribute actively for a year. Trusted contributors can contact
+team leads or will be contacted by team leads to enquire about promotion to team
+membership. Trusted contributors who are eligible for team membership do not
+have to become team members.
+
+Unlike trusted contributors, team members are expected to consider themselves
+as *maintainers* of the compiler - put otherwise, to be invested in the quality
+of the compiler codebase and overall health of the compiler team, independent
+of their own projects. Team membership is primarily intended to recognise and
+encourage participation in activities which are vital to the success of the
+compiler team and broader project.
+
+Team members are expected to participate in the ongoing maintenance tasks
+that the compiler team is responsible for (with all of the expected caveats
+for vacation time, mental health breaks, etc) - listed as
+[responsibilities][responsibilities] below. However, not all contributors need
+to participate in these responsibilities to an equal degree. Contributors should
+participate in these tasks to the degree that they are able - volunteers are not
+expected to participate as much as those employed to work on the compiler, for
+example. It is the responsibility of the compiler team leads to ensure that the
+ongoing maintenance tasks of the team can be completed sustainably.
+
+Team members have all of the same permissions and access as trusted
+contributors. Like trusted contributors, team members are considered members of
+the Rust project as a whole and are expected to follow the spirit of the [Code
+of Conduct][coc].
+
+Like trusted contributors, after inactivity for longer than a year, a
+contributor will be moved to alumni status. Members who are no longer able to
```

Comment:
Maybe this could say "member" here to remove the possible ambiguity with regular and trusted contributors?

```suggestion
team member will be moved to alumni status. Members who are no longer able to
```